client/lib/abtest/active-tests.js

```diff
@@ -1,5 +1,15 @@
 /** @format */
 export default {
+	springSale30PercentOff: {
+		datestamp: '20180413',
+		variations: {
+			upsell: 50,
+			control: 50,
+		},
+		defaultVariation: 'upsell',
+		allowExistingUsers: true,
+		localeTargets: 'any',
```

Comment:
Is this needed if we only intend to run the test for EN users and the strings aren't being translated?